modules/op-matching-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,209 @@
+[id="matching-pipeline-run-using-pipelines-as-code_{context}"]
+= Matching an event to a pipeline run
+
+You can match different Git provider events with each pipeline run by using special annotations on the pipeline run. If there are multiple pipeline runs matching an event, {pac} runs them in parallel and posts the results to the Git provider as soon a pipeline run finishes.
+
+[id="pull-matching-pipeline-run-using-pipelines-as-code_{context}"]
+== Matching a pull event to a pipeline run
+
+You can use the following example to match the `pipeline-pr-main` pipeline run with a `pull_request` event that targets the `main` branch:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-pr-main
+annotations:
+  pipelinesascode.tekton.dev/on-target-branch: "[main]" <1>
+  pipelinesascode.tekton.dev/on-event: "[pull_request]"
+# ...
+----
+<1> You can specify multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following:
```

Comment:
```suggestion
<1> You can specify multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following items:
```
or whatever word would be better - per [IBM guidance on lead-in wording](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording)